packages/ui-scripts/lib/fix-publish.js

```diff
@@ -40,13 +40,18 @@ const { createNPMRCFile } = require('./utils/npm')
 
 try {
   const pkgJSON = getPackageJSON()
-  // optional version argument
-  // ui-scripts --fix-publish 5.12.2
+  // Arguments
+  // 1: version to publish. If current version, use 'latest' otherwise e.g.: 8.1.3
+  // 2: publish type. defaults to latest. If set to 'maintenance', it will publish with vx_maintenance tag
+  // e.g.: ui-scripts --fix-publish 5.12.2 maintenance
+  const releaseVersion =
```

Comment:
I'm not sure this logic is needed, because we either say:
- `ui-scripts --fix-publish`, that fixes the current latest one
- `ui-scripts --fix-publish 8.1.3`, that fixes a specific version (on master)
- `ui-scripts --fix-publish 7.5.4 maintenance`, that fixes a maintenance branch version

Reply:
After careful (and lengthy) consideration together, we decided, that this solution is sufficient